src/components/StatusIcon/StatusIcon.js

```diff
@@ -37,13 +37,17 @@ export default class StatusIcon extends Component {
 
     /** @type {string} Status. */
     status: oneOf(STATUS),
+
+    /** @type {string} icon aria label. */
+    iconLabel: string,
```

Comment:
Would it be possible to call this `iconDescription` to maintain consistency with Carbon? https://github.com/carbon-design-system/carbon/search?q=iconDescription

Reply:
Sure thing!